deploy-to-aks/action.yml

```diff
@@ -0,0 +1,147 @@
+name: Deploy to AKS environment
+description: Deploy a docker image to an AKS environment
+
+inputs:
+  azure-credentials:
+    required: true
+  environment:
+    description: Environment to deploy to
+    required: true
+  github-token:
+    description: Default Github token retrieved via secrets.GITHUB_TOKEN or PAT with permission to the repository (Required)
+    required: true
+  pr-number:
+    description: PR number for the review app
+    required: false
+  sha:
+    description: Commit sha to be deployed
+    required: true
+  slack-webhook:
+    required: false
+  db-seed:
+    description: run seed command after a review deployment
+    type: boolean
+    required: false
+    default: false
+  smoke-test:
+    description: Enable smoke test after deployment
+    type: boolean
+    required: false
+    default: false
+  healthcheck:
+    description: Health check path
+    required: false
+  terraform-url-output:
+    description: terraform output that contains the app urls
+    required: false
+    default: 'external_urls'
+  terraform-base:
+    description: path to the terraform files
+    required: false
+    default: 'terraform/application'
+  terraform-provider:
```

Comment:
Seems easy to make the same for all repos. Can we hardcode to terraform.tf and fix repos going forward?